application/classes/Ushahidi/Validator/Form/Update.php

```diff
@@ -31,12 +31,27 @@ public function __construct(FormRepository $repo)
 
 	protected function getRules()
 	{
+
+			// Always check validation for name and description
+			$fullData = $this->validation_engine->getFullData();
```

Comment:
This whole block shouldn't be required. By design we generally only validate changes to an entity.